examples/src/examples/graphics/mesh-displacement.tsx

```diff
@@ -0,0 +1,301 @@
+import React from 'react';
+import * as pc from '../../../../';
+
+import { BindingTwoWay, LabelGroup, Panel, SliderInput, SelectInput, BooleanInput } from '@playcanvas/pcui/react';
+import { Observer } from '@playcanvas/observer';
+class MeshDisplacementExample {
+    static CATEGORY = 'Graphics';
+    static NAME = 'Mesh Displacement';
+    static WEBGPU_ENABLED = false;
+
+    controls(data: Observer) {
+        return <>
+            <Panel headerText='Displacement Effect'>
+                <LabelGroup text='Enabled'>
+                    <BooleanInput id='displacment-effect' binding={new BindingTwoWay()} link={{ observer: data, path: 'script.useDisplacement.enabled' }} />
+                </LabelGroup>
+                <LabelGroup text='Height Map Factor'>
+                    <SliderInput binding={new BindingTwoWay()} link={{ observer: data, path: 'script.useDisplacement.heightMapFactor' }} min={-2.0} max={2.0}/>
+                </LabelGroup>
+                <LabelGroup text='Displacement Offset'>
+                    <SliderInput binding={new BindingTwoWay()} link={{ observer: data, path: 'script.useDisplacement.displacementOffset' }} min={-2.0} max={2.0}/>
+                </LabelGroup>
+            </Panel>
+            <Panel headerText='Point light'>
```

Comment:
I have not tried running it (yet), but it seems the point light control is not related to the displacement - ideally please strip out unrelated things to keep the example well focused.